Doc/library/sys.rst

```diff
@@ -926,7 +926,8 @@ always available.
 
     A list of :term:`meta path finder` objects that have their
     :meth:`~importlib.abc.MetaPathFinder.find_spec` methods called to see if one
-    of the objects can find the module to be imported. The
+    of the objects can find the module to be imported. By default, it holds entries
+    to handle the standard kinds of modules (.py files, extension modules...). The
```

Comment:
```suggestion
   that implement Python's default import semantics. The
```

Reply:
@brettcannon Do you think we should add examples after that. Like:

> that implement Python's default import semantics. (BuiltinImporter, FrozenImporter, and PathFinder)

Reply:
No, as that's too easy of a thing to forget to update. Plus there's no guarantee that every interpreter will set up `sys.meta_path` the same.